### DISCUSSION TITLE
Is it possible to retrieve the references?

### DISCUSSION THREAD

**Discussion by MoritzImendoerffer** [scholarly-python-package/scholarly#278] [Q&A] [answered]
Hi, first of all thank you for this great package. I was wondering, if it is possible to retrieve the references (the articles an article is citing) from google scholar somehow? As I understand the documentation, it is possible to get the articles which are citing an article.

**Comment by arunkannawadi** [comment, marked as answer]
Google Scholar doesn't have the feature of listing references, only citations. So as a scraper, `scholarly` can't find references either. I believe Google Scholar doesn't list that because it is typically easy to find the references from the publications themselves, even if they are behind a paywall. Wondering what your use-case for finding references are.

**Comment by MoritzImendoerffer** [comment]
I wanted to generate a citation map. Playing around, identifying key
players per subject with whom they are collaborating etc.

Arun Kannawadi ***@***.***> schrieb am Di., 31. Aug. 2021,
16:04:
